catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/CsexraAttribute.java

```diff
@@ -31,27 +31,71 @@
 import org.codice.imaging.nitf.core.tre.Tre;
 
 import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.BasicTypes;
 
 /**
  * TRE for "Exploitation Reference Data"
  */
-class CsexraAttribute extends NitfAttributeImpl<Tre> {
+public class CsexraAttribute extends NitfAttributeImpl<Tre> {
 
     private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
 
-    static final String SNOW_DEPTH_CAT = "SNOW_DEPTH_CAT";
+    static final String SNOW_DEPTH_CAT_NAME = "SNOW_DEPTH_CAT";
```

Comment:
is it intentional all of these constants are package-private? How would an integrator get the attribute for SNOW_DEPTH_CAT?